components/tikv_util/src/security.rs

```diff
@@ -20,6 +21,7 @@ pub struct SecurityConfig {
     #[serde(skip)]
     pub override_ssl_target: String,
     pub cipher_file: String,
+    pub x509_common_names: HashSet<String>,
```

Comment:
should be consistent with PD/TIDB naming of the same config.

Reply:
Fix